packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.test.tsx

```diff
@@ -180,3 +180,33 @@ it('should render borderless', () => {
     'iui-borderless',
   );
 });
+
+it('should render disabled', () => {
+  const { container } = render(
+    <ExpandableBlock title='test title' disabled>
+      content
+    </ExpandableBlock>,
+  );
+  expect(container.querySelector('.iui-expandable-block')).toHaveClass(
+    'iui-disabled',
+  );
+});
+
+it('should not trigger onToggle when clicked  on header', () => {
+  const onToggleMock = jest.fn();
+  const { container } = render(
+    <ExpandableBlock
+      title='test title'
+      onToggle={onToggleMock}
+      disabled
+      isExpanded={false}
+    >
+      test content
+    </ExpandableBlock>,
+  );
+
+  const header = container.querySelector('.iui-title') as HTMLElement;
+  expect(header).toBeTruthy();
+  fireEvent.click(header);
+  expect(onToggleMock).toHaveBeenCalledTimes(0);
+});
```

Comment:
suggestions to improve unit test:
- combine these two tests
- instead of querying by classname, try querying by role or text
- check for `aria-disabled` instead of `iui-disabled`
- also check for `aria-expanded`

```js
it('should respect disabled prop', () => {
  const onToggleMock = jest.fn();
  const { getByRole } = render(
    <ExpandableBlock title='test title' onToggle={onToggleMock} disabled>
      test content
    </ExpandableBlock>,
  );

  const header = getByRole('button');
  expect(header).toBeDisabled();
  fireEvent.click(header);
  expect(onToggleMock).not.toHaveBeenCalled();
  expect(header).toHaveAttribute('aria-expanded, 'false');
});
```

Reply:
What method do I use to check for aria-disabled?

Reply:
`toBeDisabled()` should work, but if not, then you can use `toHaveAttribute('aria-disabled', 'true')`

Reply:
Added suggested unit test

Reply:
unfortunately it is failing so you'll have to change it to `toHaveAttribute`. https://github.com/iTwin/iTwinUI/actions/runs/5148140855/jobs/9269498639

make sure it passes locally before committing